### PR TITLE
some-fixes

### PR DESCRIPTION
**Pull request by wissog**
converted classes to functions, deleted unused etc

### CODE REVIEW

**Review by gshilin-sdb — CHANGES_REQUESTED** [review]

**Comment by gshilin-sdb** [comment]
Why are you removing `SameCollection` stuff?

**Comment by wissog** [comment]
SameCollection is not in use, we made Recommended functionality instead
You still want to keep it?
בברכה,
גבריאל ויסוצקי
נייד: 054-5606-798

**Comment by gshilin-sdb** [comment]
No, I'm not.

File: src/components/Filters/components/Date/YearMonthForm.js [hunk 1]
```diff
@@ -1,81 +1,59 @@
...
-          onChange={this.handleYearChange}
-        />
-      </span>
-    );
-  }
+      <Dropdown
+        compact
+        inline
+        scrolling
+        options={years.map(yea => ({ text: yea, value: yea }))}
```
[the remark this answers is not in the document]
Comment (wissog) [reply src/components/Filters/components/Date/YearMonthForm.js]: ok

**Review by gshilin — APPROVED** [review]

**Pull request merged**